app/index.js

```diff
@@ -165,6 +167,9 @@ app.on('ready', () => installDevExtensions(isDev).then(() => {
       // notify renderer
       win.webContents.send('config change');
 
+      // Update Vibrancy
+      win.setVibrancy(toElectronVibrancy(cfg_.vibrancyType));
```

Comment:
Seems the background color change moved elsewhere, but I'm having a hard time finding it - did it go into the reducer? I'm not sure I can put this into state; also not sure it belongs. Would appreciate advice!